### PR TITLE
LPS-110898 Viewport Size Selector

### PR DESCRIPTION
**Pull request by veroglez**

### CODE REVIEW

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/Toolbar.scss [hunk 1]
```diff
@@ -45,13 +55,23 @@
...
 				padding-right: 0;
 
 				@include media-breakpoint-up(lg) {
 					order: 0;
 				}
+
+				.nav-item:last-child {
+					margin-left: 16px;
+					@include media-breakpoint-up(lg) {
+						left: calc(50% + 47px);
```
Comment (p2kmgcl) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/Toolbar.scss]: #cuarentaYSiente

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/config/constants/availableViewportSizes.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+export const AVAILABLE_VIEWPORT_SIZES = {
```
Comment (p2kmgcl) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/config/constants/availableViewportSizes.js]: We should try to move this to DisplayContext :smile:

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/reducers/index.js [hunk 3]
```diff
@@ -51,6 +52,7 @@ const combinedReducer = (state, action) =>
 		pageContents: pageContentsReducer,
 		permissions: permissionsReducer,
 		reducers: baseReducer,
+		selectedViewportSize: viewportReducer,
```
Comment (p2kmgcl) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/reducers/index.js]: I'm going to (_cause lots of conflicts_) rename all reducers to match the state they are managing (ex. `resolvedCommentsReducer` -> `showResolvedCommentsReducer`)

**Comment by jkappler** [comment]
Just started reviewing :)

**Comment by jkappler** [comment]
see: https://github.com/p2kmgcl/liferay-portal/pull/601

**Pull request closed without merging**